endpoints/openrtb2/amp_auction_test.go

```diff
@@ -49,6 +49,7 @@ func TestGoodAmpRequests(t *testing.T) {
 				"imp-with-stored-resp.json",
 				"gdpr-no-consentstring.json",
 				"gdpr.json",
+				"aliased-buyeruids-case-insensitive.json",
```

Comment:
I see that you have two test case files that you added for this PR. Would it mean that it also needs to be added in this function?

Reply:
Good catch! Added it in [07427d6](https://github.com/prebid/prebid-server/pull/3152/commits/07427d696b9bd7d5e38c1fc3915965d69cb34fc5)